tests/helpers.js

```diff
@@ -1364,4 +1572,64 @@ describe('helpers', function () {
     assert.equal(expect, result.type);
     done();
   });
+
+  it('collectSuffixExtensions - no extensions', function () {
+    var ruleset = gonzales.parse(['',
+      '.a {\n',
```

Comment:
@sasstools/sass-lint-contributors have we got any suggestions here or are we ok with this?